packages/common/y-provider/package.json

```diff
@@ -31,6 +31,7 @@
     "yjs": "^13.6.10"
   },
   "peerDependencies": {
+    "@blocksuite/global": "*",
```

Comment:
This was also used at runtime without being specified. Added it as a peer dep since it didn't have any direct runtime deps before.